docs/source/developer/literal.rst

```diff
@@ -0,0 +1,82 @@
+.. _developer-literally:
```

Comment:
this file needs line wrapping at 80 chars, the github suggest thing strikes again.

Reply:
fixed